### DISCUSSION TITLE
Copilot stops C++ code generation right before #if statements in VS

### DISCUSSION THREAD

**Discussion by kaetemi** [community/community#47934] [Copilot Conversations]
### Select Topic Area

Bug

### Body

See video.

https://user-images.githubusercontent.com/1581053/219927275-8c9235f5-153c-4fef-9412-f21103c95e14.mp4

**Comment by AsafMeizner** [comment]
there isn't really a problem... if you don't like it like this the only solution is to disable it...

**Reply by kaetemi** [reply]
Rude.

**Comment by kaetemi** [comment]
Also keeps stopping halfway a generated function call whenever any ip address appears in a string. Makes generating test cases for ip address parsing a bit unfeasible.

Stopping code gen halfway a function or halfway a block is a bit messy at least anyway. :)
